src/js/index.js

```diff
@@ -1,52 +1,78 @@
-const title = document.getElementById('title');
-const author = document.getElementById('author');
-const btn = document.getElementById('btn');
-const storageList = document.getElementById('storage_list');
-if (!localStorage.getItem('count')) {
-  localStorage.setItem('count', 0);
-}
-if (!localStorage.getItem('books')) {
-  localStorage.setItem('books', JSON.stringify([]));
+class Ls {
```

Comment:
- [ ] **[OPTIONAL] Please consider using more descriptive names in your code.**

The name `Ls` makes it difficult to understand the code. You can name the class something like `Books` for example.